Objects/Enemy/Enemy.gd

```diff
@@ -1,22 +1,32 @@
 extends KinematicBody2D
 
 onready var Vehicle = get_parent().get_node("Vehicle")
+onready var Player = get_parent().get_node("Player")
+onready var ray = $RayCast2D
+var rayLength = 16
 
 export (int) var speed = 40	# Max speed of enemy movement
 export (bool) var hasPart = false # Is enemey holding a vehicle part?
 
 # Types: 0 => nothing; 1 => small; 2 => medium; 3 => large
```

Comment:
We can actually use `ENUM` for this. It would look like:
```
ENUM WEIGHTS {
SMALL,
MEDIUM,
LARGE
}
```

And would assign values to those different options accordingly (I believe based on index + 1).

Then you could reference that value with `self.WEIGHTS.{weight name}`, and we could change that integer value from one centralized place.

Reply:
👍